helm/templates/rbac.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sablier-{{ .Release.Name }}
+rules:
+  - apiGroups:
+      - apps
+      - ""
+    resources:
+      - deployments
+      - deployments/scale
+      - statefulsets
+      - statefulsets/scale
+    verbs:
+      - patch # Scale up and down
+      - get # Retrieve info about specific dep
+      - update # Scale up and down
+      - list # Events
+      - watch # Events
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: sablier-{{ .Release.Name }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: sablier-{{ .Release.Name }}
+subjects:
+  - kind: ServiceAccount
+    name: sablier
```

The `sa` name doesn't match the one created in the other file

```suggestion
    name: sablier-{{ .Release.Name }}
```